docs/words.tex

```diff
@@ -378,8 +378,10 @@ \section{Disk I/O}
 \item[required ( filenameptr filenamelength -- )] Like included, except that load is skipped if the file is already loaded.
 \item[loadb ( filenameptr filenamelength dst -- endaddr )] Load binary block to dst. Returns 0 on failure, otherwise address after last written byte.
 \item[saveb ( start end filenameptr filenamelength -- )] Save binary block.
-\item[device ( device\# -- )] Switches the active device. 8 to 11 are valid device\#'s.
+\item[device ( device\# -- )] Switches the active device. 8 to 11 are valid device \#'s.

```

Comment:
I realize now that a better change would be to delete the sentence "8 to 11 are valid device\#'s."  It is misleading, other device #'s are valid too.

Reply:
DEVICE is somewhat less ambiguous.